README.md

```diff
@@ -1,2 +1,53 @@
-# _template
-Template repo with README and CI settings
+# LV SJPG DECODER 
+- (first commit)
+
+# Overview:
+  - SJPG is 'split-jpeg' which is a bundle of small jpeg fragments with an sjpg header.
+  - SJPG size will be almost comparable to the jpg file or might be a slightly larger.
+  - lv_sjpg can open and handle multiple files at same time.
+  - File read from disk (fread) and c-array is implemented.
+  - SJPEG frame fragment cache enables fast fetching of lines if availble in cache.
+  - By default the image cache will be xres * 2 * 16 bytes (can be modified)
+  - Currently only 16 bit image format is supported (to do)
+# Sample code:
+```c
+lv_obj_t * img1;
+lv_obj_t * img2;
+
+void lv_demo_sjpeg(void)
+{
+    lv_jpeg_init();
+    img1 = lv_img_create(lv_scr_act(), NULL);
+    img2 = lv_img_create(lv_scr_act(), NULL);
+    lv_img_set_src(img1,  &wallpaper);            //image 1 from c array
+    lv_img_set_src(img2,  ".\\small_image.sjpg")  //image 2 from file system (fopen)
```

Comment:
Finally, we should use `lv_fs` to allow handling different file systems in a transparent way. [lv_fs_if](https://github.com/lvgl/lv_fs_if) has ready to use binding to C file functions.

Reply:
Okay, got it !